lib/omniauth/strategies/auth0.rb

```diff
@@ -86,7 +86,7 @@ def client
       def authorize_params
         params = super
         parsed_query = Rack::Utils.parse_query(request.query_string)
-        %w[connection connection_scope prompt].each do |key|
+        %w[connection connection_scope prompt screen_hint].each do |key|
```

Comment:
what is this line doing, allowing these keys to be taken from the URL query and setting them on the /authorize call?